server/src/index.ts

```diff
@@ -86,12 +114,15 @@ const typeDefs = gql`
     databases: [Database!]!
     dokkuPlugins: DokkuPluginResult!
     appLogs(name: String!): AppLogsResult!
+    envVars(name: String!): EnvVarsResult!
```

Comment:
I just saw that but actually we will have access to appId instead of app name, will be the same `appLogs`.